pkgs/development/python-modules/posting/default.nix

```
@@ -0,0 +1,87 @@
+{ pkgs ? import <nixpkgs> {} }: let
```

```suggestion
{ buildPythonPackage, hatchling }: let
```
Do not use `import <nixpkgs>` in nixpkgs itself, instead list all the required arguments.